Tests/Marketplace/search_and_install_packs.py

```diff
@@ -256,8 +264,93 @@ def install_nightly_packs(client: demisto_client, host: str, prints_manager: Par
             prints_manager.execute_thread_prints(thread_index)
 
 
-def install_packs(client, host, prints_manager, thread_index, packs_to_install, request_timeout=999999,
-                  private_install=False, is_nightly=False):
+def install_testing_license(client: demisto_client, host: str, prints_manager: ParallelPrintsManager,
```

Comment:
we don't need the testing license because the AMI should use the dev license. sync with @hod-alpert